bin/docker/kafka_bridge_config_generator.sh

```diff
@@ -122,6 +122,7 @@ HTTP_PROPERTIES=$(cat <<-EOF
 http.enabled=${KAFKA_BRIDGE_HTTP_ENABLED}
 http.host=${KAFKA_BRIDGE_HTTP_HOST}
 http.port=${KAFKA_BRIDGE_HTTP_PORT}
+http.enable_cors=${KAFKA_BRIDGE_§CORS_ENABLED}
```

Comment:
Do we need to aslo add the other options as in `config/application.properties`?

Reply:
right, added in last commit c253b6592b9b7317bd2a214c618a601477c20992